ports/mimxrt/tusb_port.c

```diff
@@ -25,12 +25,17 @@
  */
 
 #include "tusb.h"
+#include "mphalport.h"
 
 #ifndef MICROPY_HW_USB_VID
 #define MICROPY_HW_USB_VID (0xf055)
 #define MICROPY_HW_USB_PID (0x9802)
```

Comment:
If we are touching the tusb config then please add an `#ifndef MICROPY_HW_USB_PID` as well. In my point of view we should be able to configure both VID and PID individually.

Like:
```
#ifndef MICROPY_HW_USB_VID
#define MICROPY_HW_USB_VID (0xf055)
#endif
#ifndef MICROPY_HW_USB_PID
#define MICROPY_HW_USB_PID (0x9802)
#endif
```

Reply:
Olimex have the Christmas party right now, but you can look at my fork. I made some other changes

Reply:
Hey no worries - by all means enjoy your Christmas party. I just think that this suggestion is a worthy improvement which you should consider bringing in with your PR.